tests/unit/test_tools.py

```diff
@@ -43,3 +48,41 @@ def test_environment_action_invalid_env(monkeypatch):
             dest='PLAT',
             choices=choices,
             default='manylinux1')
+
+
+def _write_test_permissions_zip(path):
+    source_zip_xz = Path(__file__).parent / "test-permissions.zip.xz"
```

Comment:
Why is this file compressed twice?

Reply:
the zip is 215kB, with xz compression, goes down to 8kB